waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -45,3 +45,22 @@ export const invalidateQueriesUsing = (resources) => {
     queryClient.invalidateQueries(queryCacheKey)
   }
 }
+
+export function removeQueries() {
+  // TODO(filip): We are currently removing all the queries, but we should
+  // remove only non-public, user-dependent queries - public queries are
+  // expected not to change in respect to the currently logged in user.
+  queryClient.removeQueries()
+}
+
+// TODO(filip): We are currently invalidating and removing  all the queries, but
+// we should remove only the non-public, user-dependent ones.
+export function invalidateAndClearQueries() {
```

Comment:
Why is it called invalidateAndClear, and not invalidateAndRemove?

Reply:
:sweat_smile: